client/src/components/chat/chat-item.tsx

```diff
@@ -53,14 +54,17 @@ export interface ChatItemData extends ChatMsg {
   name: string;
   color: string;
 }
+
 export function ChatItem(props: {
   message: ChatItemData;
   i: number;
   styles: StyleProps;
   setAnswerVisibility: (show: boolean) => void;
   visibility: boolean;
+  chatProps: ChatProps;
```

Comment:
don't add the whole chatProps here...reading below I think all you want from ChatProps `displayMentorNames?: boolean`, so just include that single prop here

Reply:
done